File: LTUDDD_TLU/app/src/main/java/com/example/ltuddd_tlu/tuan2/tuan22_java/tuan22MainActivity.java
package com.example.ltuddd_tlu.tuan2.tuan22_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import com.example.ltuddd_tlu.R;

public class tuan22MainActivity extends AppCompatActivity {

    ListView lv;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tuan22_main2);

        lv = findViewById(R.id.tuan22Listview);

        //nguon du lieu
        String[] arr = new String[]{"Mon 1", "Tuan 2", "Mon 3", "Mon 4", "Mon 5"};

        //adapter
        ArrayAdapter<String> adapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, arr);

        //hien thi len listView
        lv.setAdapter(adapter);

    }
}

File: LTUDDD_TLU/app/src/main/java/com/example/ltuddd_tlu/tuan2/tuan22_KL/tuan22MainActivity.kt
package com.example.ltuddd_tlu.tuan2.tuan22_KL

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import com.example.ltuddd_tlu.R

class tuan22MainActivity : AppCompatActivity() {

    var txt1: EditText? = null
    var txt2: EditText? = null
    var btn1: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tuan22_main)

        txt1 = findViewById(R.id.tuan22txt1)
        txt2 = findViewById(R.id.tuan22txt2)
        btn1 = findViewById(R.id.tuan22Btn1)
        btn1!!.setOnClickListener(View.OnClickListener {
            var i = Intent(this@tuan22MainActivity,tuan22SecondActivity::class.java)
            i.putExtra("so1",txt1!!.text.toString())
            i.putExtra("so2",txt2!!.text.toString())
            startActivity(i)
        })
    }
}